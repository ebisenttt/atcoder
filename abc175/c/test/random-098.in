678193757640426 312128040297644 552425410182786
