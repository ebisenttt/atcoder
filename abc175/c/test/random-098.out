426657062725146
